### DISCUSSION TITLE
Provide ability to copy, rename and delete workflows through GUI

### DISCUSSION THREAD

**Discussion by gaberger** [mikub/titanoboa#32] [General]
It doesn't seem this capability is currently available.

**Comment by mikub** [comment]
Yep, this should get added at some point. 

Currently cloning might be done simply by creating a new workflow and copy/pasting the original edn there (plus tweaking the name so as it is the same as the new one).

Deleting needs to be thought through though - the philosophy behing Titanoboa is strong focus on immutability hence currently deletion can be done via file system only which assumes admin access.
I would probably add deletion only when/if more complex authorization is in place - i.e. only certain role(s) could delete...
